python/ray/experimental/sgd/sgd.py

```diff
@@ -177,6 +180,33 @@ def warmup(self):
         ray.get([w.warmup.remote() for w in self.workers])
         logger.info("Warmup complete")
 
+    def save_checkpoint(self, path):
+        param_oids = [worker.get_params.remote() for worker in self.workers]
```

Comment:
Please use https://github.com/ray-project/ray/blob/master/python/ray/experimental/sgd/mnist_example.py#L115 instead.

You also don't need to fetch from each worker, just one will do.